openmetadata-ui/src/main/resources/ui/package.json

```diff
@@ -19,7 +19,7 @@
     "@okta/okta-auth-js": "^6.1.0",
     "@okta/okta-react": "^6.4.2",
     "autoprefixer": "^9.8.6",
-    "axios": "^0.21.1",
+    "axios": "^0.21.3",
```

Comment:
This change will break PUT api calls for followers, we have an open ticket for this here #1408